pages/twitterFeed/[screenName].js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Timeline } from 'react-twitter-widgets';
+export async function getServerSideProps(context) {
+
+
+  return {
+    props: {
+      screenName: context.params.screenName,
+    },
+  };
+}
+
+const TwitterFeed = ({ screenName }) => {
+  console.log(screenName);
```

Comment:
We can take out consoles unless you want them for further changes.

Reply:
gonna take it out forgot about it